grails-app/conf/au/org/emii/aatams/VisibilityControlFilters.groovy

```diff
@@ -36,15 +36,16 @@ class VisibilityControlFilters
         {
             after = { model ->
 
-                def instanceName = "${controllerName}Instance"
-                def instance = model[instanceName]
+                if (model) {
```

Comment:
I thought about removing those actions from the list of actions that match for this filter. The problem is that *sometimes* these actions return models (e.g. if you update but there's a validation error you go back to the error page with the model again).